doc/source/conf.py

```diff
@@ -78,7 +78,7 @@
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
+language = "en"
```

Comment:
Seems unrelated?

Reply:
Newer versions of sphinx generate a warning when the language isn't explicitly chosen to be english. I can remove this if you want, but I thought I'd sneak it in here since it seems too small to be its own PR.